src/cli_shared/cli/client.rs

```diff
@@ -82,6 +83,7 @@ impl Default for Client {
             data_dir: dir.data_dir().to_path_buf(),
             genesis_file: None,
             enable_rpc: true,
+            enable_metrics_endpoint: true,
```

Comment:
why exactly do you need this filed in the `client` section?

Reply:
To match the logic of `enable_rpc: true,`

Reply:
I see, I thought (dreamed) that the config is structured somehow differently.